BUILD

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+# BEGIN-INTERNAL
```

Comment:
Should these be `BEGIN-`/`END-EXTERNAL`? Alternately, can we just remove them with Copybara without the guards?

Reply:
In this case, these are comments that `pkg_npm` uses for removing npm-bound lines, rather than for Copybara.